lib/handlers.py

```diff
@@ -344,7 +344,7 @@ def _create_cocip_grid_source(t: datetime, flight_level: int) -> MetDataset:
         hor_res = 0.25
         dtype = np.float64
         longitude = np.arange(-180, 180, hor_res, dtype=dtype)
-        latitude = np.arange(-80, 80.01, hor_res, dtype=dtype)
+        latitude = np.arange(-90, 90, hor_res, dtype=dtype)
```

Comment:
This actually should be `np.arange(-90, 90.01, hor_res, dtype=dtype)` or `np.arange(-90, 90 + hor_res, hor_res, dtype=dtype)`. As written, the north pole isn't included.

Reply:
ah, right. thanks for the catch!